addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -16,7 +16,7 @@ class StockPickingBatch(models.Model):
 
     name = fields.Char(
         string='Batch Transfer', default='New',
-        copy=False, required=True, readonly=True)
+        copy=False, required=True)
```

Comment:
Mm, now that this is not readonly anymore, it makes it so the default behavior when creating a new batch picking without setting a name will result in the record always being called "New". It would be better if when the user doesn't specify a name, then it defaults to the current behavior (i.e. uses the "BATCH/0000#" where # is incremented up for each new batch). It's a bit tricky because the `name` field is required so we can't just leave it blank